ledger/participant-integration-api/src/test/suite/scala/platform/indexer/parallel/ParallelIndexerSubscriptionSpec.scala

```diff
@@ -162,6 +211,128 @@ class ParallelIndexerSubscriptionSpec extends AnyFlatSpec with Matchers {
     actual shouldBe expected
   }
 
+  behavior of "inputMapper transaction metering"
```

Comment:
as suggested earlier it would be beneficial to have targeted unit tests for the metering logic, and then we could just adapt the test here above

Reply:
Done.